tasks/main.yml

```diff
@@ -0,0 +1,77 @@
+---
+- name: Update repositories cache (apt)
+  apt:
+    update_cache: yes
+  changed_when: false
+  when: ansible_pkg_mgr == "apt"
+
+- name: Install required packages (apt)
+  apt:
+    name: clamav
+  when: ansible_pkg_mgr == "apt"
+
+- name: Update repositories cache (pacman)
+  pacman:
+    update_cache: yes
+  changed_when: false
+  when: ansible_pkg_mgr == "pacman"
+
+- name: Install required packages (pacman)
+  pacman:
+    name: clamav
+  when: ansible_pkg_mgr == "pacman"
+
+- name: Create configurations
+  template:
+    src: "{{ item }}.conf.j2"
+    dest: /etc/clamav/{{ item }}.conf
+    owner: root
+    group: clamav
+    mode: 0640
+  notify: Restart clamav
+  with_items:
+    - clamd
+    - freshclam
+
+- name: Create units
+  template:
+    src: "{{ item }}.service.j2"
+    dest: /etc/systemd/system/{{ item }}.service
+    owner: root
+    group: root
+    mode: 0644
+  register: clamd_unit
+  with_items:
+    - clamd
+    - freshclamd
+
+- name: Create tmpfiles configuration
+  template:
+    src: tmpfiles.conf.j2
+    dest: /etc/tmpfiles.d/clamav.conf
+    owner: root
+    group: root
+    mode: 0644
+  register: tmpfiles
+
+- name: Create tmpfiles
+  command: systemd-tmpfiles --create
```

Comment:
Did you try the `tempfile` module?

Reply:
he is creating systemd-tmpfile not tmp files

Reply:
Yup, both are for completely different purposes. The don't have anything in common except for the similar name